test/test_jit.py

```diff
@@ -5920,6 +5920,18 @@ def func(a, b):
         inputs = self._make_scalar_vars([-1234, 4321], torch.int64)
         self.checkScript(func, inputs, optimize=True)
 
+    def test_divmod(self):
+        def func_int(a, b):
+            # type: (int, int) -> Tuple[int, int]
+            return divmod(a, b)
+
+        def func_float(a, b):
+            # type: (float, float) -> Tuple[float, float]
+            return divmod(a, b)
+
+        self.checkScript(func_int, (1024, 10,))
```

Comment:
Can you also check with some negative inputs? Python and C++ [differ in their mod with negative numbers](https://stackoverflow.com/questions/1907565/c-and-python-different-behaviour-of-the-modulo-operation), we want to make sure it matches the Python behavior

Reply:
Also need to error check for 0 input